Activating multiple contests

MorganLove opened the pull request:
**Description**

- Uncommented and updated the code related to adding and removing contests on the multi jurisdiction flow.

**Testing**

- Updated snapshots and unskipped the test related to adding and removing contests

**Progress**

- The backend sorts the contests alphabetically by name from what I can tell, so contests don't come back in the same order they were entered in. This might confuse end users, if they put a contest as 'contest one' but then it comes back as the second contest. Not sure if this is an issue or not, @mcchilders ?

jonahkagan commented on client/src/components/MultiJurisdictionAudit/Setup/Contests/index.tsx:
```diff
@@ -108,22 +108,18 @@ const Contests: React.FC<IProps> = ({
-                            /* istanbul ignore next */
                             values.contests.length > 1 ? i + 1 : ''
                           } Info`}
                           description="Enter the name of the contest that will drive the audit."
```
I'm not sure this line of copy makes sense in the case where there are multiple targeted contests

MorganLove replied:
@mcchilders Thoughts on this?

jonahkagan commented on client/src/components/MultiJurisdictionAudit/Setup/Contests/index.tsx:
```diff
@@ -252,21 +240,28 @@ const Contests: React.FC<IProps> = ({
+                          >
+                            Remove Contest {i + 1}
+                          </FormButton>
+                        )}
```
wonder if wrapping each contest in a card would make it easier to parse for the user. right now it's a lot of fields that aren't super separated visually

MorganLove replied:
@jonahkagan There's an `<hr/>`, but yeah, it could be more visually distinct. There's likely a satisfactory wrapping component in the blueprintjs uikit that would work.

MorganLove commented:
@jonahkagan I added cards and updated the test to work with not using the `asyncActRender` so there's some stuff for you to look over before I merge it.

MorganLove commented on client/src/components/MultiJurisdictionAudit/Setup/Contests/index.tsx:
```diff
@@ -106,24 +107,20 @@ const Contests: React.FC<IProps> = ({
                     return (
                       /* eslint-disable react/no-array-index-key */
-                      <React.Fragment key={i}>
+                      <Card key={i}>
```
@jonahkagan Great catches! I've fixed those now.
